client/src/client.rs

```diff
@@ -387,9 +389,9 @@ impl QueryRequest {
 
         match self.request {
             iroha_data_model::query::QueryRequest::Query(query_with_params) => builder
-                .params(Vec::from(query_with_params.sorting))
-                .params(Vec::from(query_with_params.pagination))
-                .body(query_with_params.query),
+                .params(Vec::from(query_with_params.sorting().clone()))
```

Comment:
Not introduced in this PR, but the conversion of `Pagination` and `Sorting` structs to `Vec` is a bit unintuitive.. I feel like explicitly naming it something like `into_request_params`.

Maybe even make it into a `as_request_params` & make it accept a reference (it's cloned here anyways).

Reply:
opened #4017 4017